src/hashdump.c

```diff
@@ -112,7 +112,7 @@ int hashdump_main(int argc, char **argv)
 
 	ret = set_path_prefix(NULL);
 	if (!ret) {
-		free(file->filename);
+		free_string(&file->filename);
```

Comment:
This change not actually needed, but it is good for consistency

Reply:
There are several changes that were not actually needed, but I tried to convert all `free(some_string)` to `free_string(some_string)`, like you said, for consistency.